products/jbrowse-cli/src/commands/text-index.ts

```diff
@@ -192,13 +193,30 @@ export default class TextIndex extends JBrowseCommand {
     for (const config of configs) {
       const {
         adapter: { type },
-        textSearchIndexingAttributes,
-        textSearchIndexingFeatureTypesToExclude,
+        textSearchConf,
       } = config
 
-      const types = textSearchIndexingFeatureTypesToExclude || typesToExclude
+      const types: Array<string> = textSearchConf?.indexingFeatureTypesToExclude || [
```

Comment:
Here I added back the logic for modifying the include/exclude I feel as though we could review this again but from my understanding the --include/exclude tags are only to modify the configuration. If they are supposed to completely overwrite the config I can change that.

Reply:
This is similar to the other discussion, I think if we have 

config > --exclude > defaults

Then it makes a little more sense. Also currently, with this PR, the excludes and includes are both put into the same array. I think probably they should be stored in separate arrays

Reply:
Right now the include/exclude just modify whatever we pull from the config. Say configArray = [CDS, exon], include = [CDS] and exclude = [transcript]. The way it is now would modify configArray so **include** would **remove** CDS from configArray while **exclude** would **add** transcript to configArray leaving it to equal [exon, transcript]. I feel like our overall plan for --include/exclude wasn't completely planned out so I am just going off of how Teresa and I interpreted it. 

I feel like something that would be easier is just having --exclude act like --attributes where it overwrites the config array if it's specified. This would also follow the same flow of someone with an aggregate index wanting everything indexed with the same data.

Reply:
The current concept of include/exclude seems confusing so Im completely fine with changing its behavior to something like --attributes